ratis-client/src/main/java/org/apache/ratis/client/impl/ClientProtoUtils.java

```diff
@@ -18,6 +18,7 @@
 package org.apache.ratis.client.impl;
 
 import java.nio.ByteBuffer;
+import java.util.Objects;
```

Comment:
There is a checkstyle warning
[235](https://github.com/apache/ratis/runs/5218732837?check_suite_focus=true#step:4:235)
```
Error:  src/main/java/org/apache/ratis/client/impl/ClientProtoUtils.java:[21,8] (imports) UnusedImports: Unused import - java.util.Objects.
```